src/main/java/it/smartcommunitylab/aac/webauthn/service/WebAuthnRpService.java

```diff
@@ -160,15 +168,18 @@ private String convertTransportsToString(Set<AuthenticatorTransport> transports)
         return String.join(",", result);
```

Comment:
always avoid custom string join in favour of StringUtils.collectiontocommadelimitedstring...

Reply:
Fixed in 674587d6d918161c1fd98065e2cef0484a6f55da